webviz_subsurface/_assets/js/clientside_functions.js

```diff
@@ -9,5 +9,12 @@ window.dash_clientside = Object.assign({}, window.dash_clientside, {
       */
       return { data: data, layout: layout };
     },
+    get_client_height: function (_triggered) {
+      /*
+          Can be used in a dash callback to get the clientHeight
+          in pixels
+     */
+      return document.documentElement.clientHeight
+    }
   },
 });
```

Comment:
We don't need to do it in this PR, but if this function is to remain in `webviz-subsurface, we might want to prefix the file with `webviz_subsurfacee`, as well as the function names, in order to prevent namespace collisions with other plugin projects doing similar things.

Reply:
Agreed. I can make a separate PR for it.